### ISSUE TITLE
"Software Updates Failed" on Rawhide

### ISSUE THREAD

**Issue by orowith2os** [fedora-silverblue/issue-tracker#486]
This issue tracker is intended only for Silverblue specific issues. We would like to ask you to try to reproduce the issue on a relevant Fedora Workstation release. If you will be able to reproduce there, then please report it in [Red Hat Bugzilla](https://bugzilla.redhat.com/) (see [How to file a bug](https://docs.fedoraproject.org/en-US/quick-docs/howto-file-a-bug/)) or in upstream (preferred for GNOME projects) and not in this issue tracker.

**Describe the bug**
When rebasing to Rawhide, the process will execute as normal. However, upon loading the shell, GNOME Software will complain that "Software Updates Failed", but it clearly did not. 

**To Reproduce**
1. Rebase to Rawhide
2. Log in

**Expected behavior**
Software wouldn't show a notification saying "Software Updates Failed"

**Screenshots**
![image](https://github.com/fedora-silverblue/issue-tracker/assets/93224879/75cde441-d3c6-4319-9135-d8b5409476a6)
![image](https://github.com/fedora-silverblue/issue-tracker/assets/93224879/38e26b3a-69c1-4138-b28a-6101ec63bc72)

**OS version:**
```
State: idle
BootedDeployment:
● fedora:fedora/rawhide/x86_64/silverblue
                  Version: Rawhide.20230813.n.0 (2023-08-13T05:51:38Z)
               BaseCommit: e90589964ddb1fe94ad1ad175038186ab7b7a666e2ccb141692520c1b5351ec7
             GPGSignature: Valid signature by 115DF9AEF857853EE8445D0A0727707EA15B79CC
      RemovedBasePackages: firefox firefox-langpacks 116.0.2-1.fc39
          LayeredPackages: corectrl fish gnome-tweaks langpacks-en langpacks-en_GB sysprof
                           tor valgrind
            LocalPackages: opentabletdriver-0.6.0.4-1.x86_64

```

**Additional context**
Needs testing to ensure the issue doesn't happen on F38 and F39. Also need to get logs from `journalctl`.

**Comment by orowith2os** [comment]
I'm unable to test at the moment due to https://github.com/coreos/rpm-ostree/issues/4533.

**Comment by frandavid100** [comment]
It also happens on F39:

![Captura desde 2023-08-14 09-52-12](https://github.com/fedora-silverblue/issue-tracker/assets/694794/668a0269-5f81-478f-8d92-4ff54ff8c9db)
![Captura desde 2023-08-14 09-53-36](https://github.com/fedora-silverblue/issue-tracker/assets/694794/ae2004b1-4c5e-4b05-b369-eca122307b19)

Rebasing to F38 seems to work fine, though.

**Comment by travier** [comment]
Can you provide as much logs from the journal as possible? Thanks

**Comment by travier** [comment]
The following will get us the logs from the current boot:
```
journalctl -b > journal.log
```

As this is all the logs from your system, you might want to make sure that they don't contain sensitive/private information before posting it here.

**Comment by frandavid100** [comment]
> The following will get us the logs from the current boot:
> 
> ```
> journalctl -b > journal.log
> ```
> 
> As this is all the logs from your system, you might want to make sure that they don't contain sensitive/private information before posting it here.

Thanks, [here](https://pastebin.com/Z1x0YdH0)'s what that command returned. I hope it's useful.

**Comment by travier** [comment]
There are no mention of the error message in the logs so I don't know where that comes from.

**Comment by frandavid100** [comment]
Is there anything I can do? I keep getting the error message every time I reboot... although that aside, the system seems to work properly now, which it didn't a couple days ago.

![Captura desde 2023-08-16 19-51-49](https://github.com/fedora-silverblue/issue-tracker/assets/694794/61809073-1ecd-49e5-995f-c75bf498c409)

**Comment by travier** [comment]
Maybe try and run gnome-software in a terminal to get more logs? As I said above, you need to report that to GNOME Software developers upstream: https://gitlab.gnome.org/GNOME/gnome-software

**Comment by bbhtt** [comment]
This was fixed in fwupd 1.9.4 https://github.com/fwupd/fwupd/issues/6101
